src/index.html

```diff
@@ -4,11 +4,226 @@
     <title>John Roberts' CV</title>
     <link rel="stylesheet" href="./css/index.scss" />
+<style>
+
+h1 {
+font-size: 80px; 
+text-align: center;
+}
+
+h2 {
+  text-align: left;
+}
+
+
+img {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+}
+img {
+  opacity: 1.0;
+  filter: alpha(opacity=100); /* For IE8 and earlier */
+}
+
+img:hover {
+  opacity: 0.5;
+  filter: alpha(opacity=60); /* For IE8 and earlier */
+}
+.row .col {
+  float: left;
+  width: 13%;
+  border: 10px solid #fff;
+}
+
+
+table,td,th
+  {
+  border:0px solid black;
+  }
+
+table
+  {
+  width:100%;
+  }
+
+th
+  {
+  height:50px;
+  }
+td
+{
+text-align:left;
+}
+label{float: left;width: 80px;line-height:25px;} 
+form{margin:0px} 
+input{width: 180px;border:1px solid #808080;} 
+textarea{width: 250px;height: 150px;} 
+.main{
+ text-align: center;
+border-radius: 20px;
+width: 1500px;
+height: 350px;
+margin: auto;
+position: absolute;
+  top: 0;
+left: 0;
+ right: 0;
+ bottom: 0;
+}
+</style>
   </head>
   <body>
-    <div id="main"></div>
+    <div class="main">
+    
     <script src="./js/index.js"></script>
+    <h1>John Roberts</h1>
+    <h2>Home</h2>
+    <button style="margin-left: 1050px">DOWNLOAD MY FULL RESUME</button>
+    <img SRC="https://static.wixstatic.com/media/86f442ff0674441faadc2994b1b33e77.jpg/v1/crop/x_130,y_160,w_1320,h_1311/fill/w_248,h_244,al_c,q_80,usm_0.66_1.00_0.01/86f442ff0674441faadc2994b1b33e77.webp"  class="img-circle" alt="Jhon">
+    <p style="font-size:20pt;color:DodgerBlue;text-align:center">BUSINESS EXPERT<p>
+    <div id="myCarousel" class="carousel slide" data-ride="carousel">
+        <!-- Indicators -->
+        <ol class="carousel-indicators">
+          <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+          <li data-target="#myCarousel" data-slide-to="1"></li>
+          <li data-target="#myCarousel" data-slide-to="2"></li>
+          <li data-target="#myCarousel" data-slide-to="3"></li>
+        </ol>
+        <!-- Wrapper for slides -->
+      <div id="divcss5" class="carousel-inner">
+      <div class="item active">
+      <h2>ABOUT ME</h2>
+      <div class="row">
+      <div class="col"><img SRC="https://static.wixstatic.com/media/aa0402eb9ba2430d9d0620b59556efca.png/v1/fill/w_32,h_32,al_c,q_80,usm_0.66_1.00_0.01/aa0402eb9ba2430d9d0620b59556efca.webp"  class="img-circle" alt="linkedin"></div>
+      <div class="col"><img SRC="https://static.wixstatic.com/media/4057345bcf57474b96976284050c00df.png/v1/fill/w_32,h_32,al_c,q_80,usm_0.66_1.00_0.01/4057345bcf57474b96976284050c00df.webp"  class="img-circle" alt="facebook"></div>
+      <div class="col"><img SRC="https://static.wixstatic.com/media/870f97661ed14a5bb2d96ecbddec0aed.png/v1/fill/w_32,h_32,al_c,q_80,usm_0.66_1.00_0.01/870f97661ed14a5bb2d96ecbddec0aed.webp"  class="img-circle" alt="twitter"></div>
+      </div>
+     
+     <div class="item">
+     <table>
```

Comment:
indentations need to be fixed.

Reply:
Hi @yuansutina , have you follow the instruction to set up the node modules for your local development? The reason why I am asking is that if you installed node packages correctly, it should automatically format your code when your committing.

Please double-check the [instruction wiki](https://github.com/nzittraining/cv-assignment/wiki/Assignment-Requirement) and re-install your node packages. Thanks

Reply:
That is way my indentation have problem. I try to install couple times all is done, But when I running it does not work. So I used note finished my work.